source/administration/plugins.rst

```diff
@@ -53,20 +53,18 @@ Mattermost supports plugin uploads by System Admins, which allow you to customiz
 
 By default, plugin uploads are disabled on your server. To enable them, set **PluginSettings > EnableUploads** to ``true`` in your ``config.json`` file. You can disable plugin uploads anytime to control which plugins are installed on your server. This action won't disable plugins already installed on your server.
 
-Once enabled, install plugins in one of the following ways:
+Once enabled, install plugins in one of the following ways. The steps assume you previously generated a ``plugin.tar.gz`` file.
 
 1) Through System Console UI:
  - Log in to Mattermost as a System Admin.
- - Navigate to **Plugins > Management** and upload the `plugin.tar.gz` you generated above.
+ - Navigate to **Plugins > Management** and upload ``plugin.tar.gz``.
  - Click "Activate" under the plugin after it has uploaded.
 
 2) Manually:
- - Extract `plugin.tar.gz` to a folder with the same name as the plugin id you specified in ``plugin.json/plugin.json``.
+ - Extract ``plugin.tar.gz`` to a folder with the same name as the plugin id you specified in ``plugin.json/plugin.json``.
```

Comment:
Where is the `plugin.json/plugin.json` file located?

Reply:
Updated, let me know if that helps clarify its location and overall directory structure.

Reply:
@icelander Do the updates look ok?